### PR TITLE
[TR] PIM-4750: Add default thumbnail for unknown file types

### PR DESCRIPTION
**Pull request by willy-ahva**

### CODE REVIEW

File: cookbook/product_asset/add_asset_default_thumbnail_for_unknown_file_types.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
+How to Add a Default Thumbnail For Unknown File Types
+=====================================================
+
+Quick Overview
+--------------
+
+**This cookbook is about a feature only provided in the Enterprise Edition.**
+
+In the asset management each file type has a default image but it's possible you want to add a filetype which has not.
```
Comment (rybus) [inline cookbook/product_asset/add_asset_default_thumbnail_for_unknown_file_types.rst]: [...] image. Thus, if you add a new filetype, you might want to add a default one as well.

File: cookbook/product_asset/add_asset_default_thumbnail_for_unknown_file_types.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+Adding a default image for an unknown file type is overriding a service and a parameter in the dependency injection.
```
Comment (rybus) [inline cookbook/product_asset/add_asset_default_thumbnail_for_unknown_file_types.rst]: To add a [...], you have to override [...]

File: cookbook/product_asset/add_asset_default_thumbnail_for_unknown_file_types.rst [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+.. note::
+    Learn more about Symfony2 Dependency injection on the documentation : http://symfony.com/doc/current/components/dependency_injection/index.html.
```
Comment (rybus) [inline cookbook/product_asset/add_asset_default_thumbnail_for_unknown_file_types.rst]: Dependency Injection or dependency injection (caps at first char)

File: cookbook/product_asset/add_asset_default_thumbnail_for_unknown_file_types.rst [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+For the need of the cookbook, we'll add a default thumbnail for audio type. Currently, if an audio file is added in Akeneo this image is shown:
```
Comment (rybus) [inline cookbook/product_asset/add_asset_default_thumbnail_for_unknown_file_types.rst]: For the sake of this cookbook, [...]

File: cookbook/product_asset/add_asset_default_thumbnail_for_unknown_file_types.rst [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+.. image:: ./misc_default.png
+
+The goal is to show this one instead:
+
+.. image:: ./audio_default.png
+
+This cookbook assumes that you already created a new bundle. Its namespace will be `Acme\\CustomBundle`.
```
Comment (rybus) [inline cookbook/product_asset/add_asset_default_thumbnail_for_unknown_file_types.rst]: you already have created

File: cookbook/product_asset/add_asset_default_thumbnail_for_unknown_file_types.rst [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+        pim_enrich.guesser.file_type:
+            class: %pim_enrich.guesser.file_type.class%
+            arguments:
+                - acme_custom_file_audio: ['audio/*']
+
+Then remove Symfony cache and reinstall PIM assets:
+``php app/console cache:clear``
+``php app/console pim:install:assets``
+
+Now, for example, add a mp3 file in an asset and in the grid you'll see your new thumbnail.
```
Comment (rybus) [inline cookbook/product_asset/add_asset_default_thumbnail_for_unknown_file_types.rst]: f"or example" > "you can"

File: cookbook/product_asset/add_new_transformation.rst [hunk 7]
```diff
@@ -22,7 +22,7 @@ Then, compiler pass will inject your tagged Transformation in the registry which
 .. note::
     Learn more about registry_ pattern.
 
-This cookbook assumes that you already created a new bundle to add your new Transformation. Its namespace will be `Acme\CustomBundle`.
+This cookbook assumes that you already created a new bundle to add your new Transformation. Its namespace will be `Acme\\CustomBundle`.
```
Comment (rybus) [inline cookbook/product_asset/add_new_transformation.rst]: that you have already created [...]

**Comment by nidup** [comment]
GTM

**Pull request merged**